minifiers/minfiers.go

```diff
@@ -11,13 +11,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Package minifiers contains minifiers mapped to MIME types. This package is used
```

Comment:
This file should be named "minifiers.go" not "minfiers.go"

Reply:
I whish minfiers was a word, though  :-)